build/teamcity/internal/cockroach/nightlies/private_roachtest_impl.sh

```diff
@@ -21,4 +21,4 @@ build/teamcity-roachtest-invoke.sh \
   --cockroach-short="${PWD}/bin/cockroach-short-ea" \
   --artifacts=/artifacts \
   --artifacts-literal="${LITERAL_ARTIFACTS_DIR:-}" \
-  costfuzz/workload-replay
+  "${TESTS}"
```

Comment:
`TESTS` is exposed to the Docker container on line 10 of `build/teamcity/internal/cockroach/nightlies/private_roachtest.sh`. I've already updated the build configuration to set the environment variable.

Reply:
Why not hard-code the default so it's in code and make it get overridden if set manually in a run?

Reply:
Behaviorally it doesn't seem to make a difference, but I can make the change just to be explicit. Done.

Reply:
Cool. Thank you!

Agreed on it not making a behavioral difference currently. Though it could in the future when new tests need to be added and 23.X is already released. Also, there is the desire to keep as much configuration in the repo as possible.